### PR TITLE
The Udash framework

### PR DESCRIPTION
**Pull request by Starzu**

### CODE REVIEW

File: _data/library/scalalibs.yml [hunk 1]
```diff
@@ -100,6 +104,9 @@
       url: https://github.com/lihaoyi/autowire
       desc: Cross-compiling statically-typed Ajax calls and RPCs
       dep: '"com.lihaoyi" %%% "autowire" % "0.2.5"'
+    - name: Udash RPC
+      url: http://guide.udash.io/#/rpc
+      desc: Cross-compiling strongly typed client-server communication system for the Udash framework
```
Comment (sjrd) [inline _data/library/scalalibs.yml]: Is the RPC layer usable in any way without using the full Udash framework?

Comment (Starzu) [inline _data/library/scalalibs.yml]: Yes, it is. You can use Udash RPC with scalajs-react or widok.

Comment (sjrd) [inline _data/library/scalalibs.yml]: OK

**Comment by sjrd** [comment]
LGTM

**Pull request merged**